RecoTracker/MkFit/plugins/MkFitGeometryESProducer.cc

```diff
@@ -10,44 +11,431 @@
 
 #include "RecoTracker/MkFit/interface/MkFitGeometry.h"
 
-#include "createPhase1TrackerGeometry.h"
-
 // mkFit includes
 #include "RecoTracker/MkFitCore/interface/ConfigWrapper.h"
 #include "RecoTracker/MkFitCore/interface/TrackerInfo.h"
 #include "RecoTracker/MkFitCore/interface/IterationConfig.h"
+#include "RecoTracker/MkFitCMS/interface/LayerNumberConverter.h"
 
 #include <atomic>
 
+//------------------------------------------------------------------------------
+
+#include "DataFormats/GeometrySurface/interface/RectangularPlaneBounds.h"
+#include "DataFormats/GeometrySurface/interface/TrapezoidalPlaneBounds.h"
+//#include "Geometry/CommonDetUnit/interface/GlobalTrackingGeometry.h"
+//#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+//#include "Geometry/TrackerGeometryBuilder/interface/StripGeomDetUnit.h"
+//#include "Geometry/TrackerGeometryBuilder/interface/StripGeomDetType.h"
+//#include "Geometry/TrackerGeometryBuilder/interface/RectangularPixelTopology.h"
+#include <fstream>
+
+//------------------------------------------------------------------------------
+
 class MkFitGeometryESProducer : public edm::ESProducer {
 public:
-  MkFitGeometryESProducer(const edm::ParameterSet& iConfig);
+  MkFitGeometryESProducer(const edm::ParameterSet &iConfig);
 
-  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+  static void fillDescriptions(edm::ConfigurationDescriptions &descriptions);
 
-  std::unique_ptr<MkFitGeometry> produce(const TrackerRecoGeometryRecord& iRecord);
+  std::unique_ptr<MkFitGeometry> produce(const TrackerRecoGeometryRecord &iRecord);
 
 private:
+  struct gap_collector {
+    struct interval {
```

Comment:
style: capitalized (CamelCap) is preferred for types

Reply:
renamed